MTA-6811, 6812, 6813, 6814, 6815, 6816 CQA April Developer Lightspeed work (#357)
migtools/mta-documentation: docs/topics/developer-lightspeed/proc_configuring-developer-profile-settings.adoc

Reviewer: ```suggestion
```

I think this floating plus sign breaks Verification heading in the preview.

Author: Appended a table of profile config options here. So, I retained the plus sign to align the table to the step.

Reviewer: ```suggestion
```

Not needed between a leading sentence and a list.

Reviewer: ```suggestion

```

Not needed between items in a single list.

Reviewer: ```suggestion
* After you complete the profile configuration, close the `Get Ready to Analyze` pane. You can verify that your configuration works by running an analysis.
```

Verification calls for one or more steps, see https://redhat-documentation.github.io/modular-docs/#verification.

Reviewer: The line reads "The `Get Ready to Analyze` pane lists the following basic configurations" yet there are no configs explained?

Author: Attached the profile config options here :thinking:

= Configuring the {mta-dl-plugin} profile settings

[role="_abstract"]

> To perform repeated analyses, you can create a reusable profile configuration. You can use the profile you configure in your Integrated Development Environment (IDE).  

Reviewer 2: A more direct action for the procedure
```suggestion
To perform repeated analyses, create a reusable profile configuration. You can use the profile you configure in your Integrated Development Environment (IDE).  
```

.Prerequisites

* You completed the Solution Server configurations in Tackle custom resource if you opt to use the Solution Server.
* You opened a project in your Visual Studio Code workspace.

.Procedure

. Open the `{ProductShortName} View Analysis` page in either of the following ways:
.. Click the book icon on the `{ProductShortName}: Issues` pane of the {ProductShortName} extension.
.. Type `Ctrl + Shift + P` or `Cmd + Shift + P` on the search bar to open the Command Palette and enter `{ProductShortName}:Open Analysis View`.
. Click the settings button on the `{ProductShortName} View Analysis` page to configure a profile for your project.
+

> The `Get Ready to Analyze` pane lists the profile configuration options.

Reviewer 2: ```suggestion
The `Get Ready to Analyze` pane lists the profile configuration options:
```

|Configure label selector|A label selector filters rules for analysis based on the source or target technology.

Specify one or more target or source technologies (for example, cloud-readiness). {mta-dl-plugin} uses this configuration to determine the rules to apply to a project during analysis.

If you mentioned a new target or a source technology in your custom rule, you can type that name to create and add the new item to the list.

[NOTE]
====
You must configure either target or source technologies for running an analysis.
====
|Set rules|Enable default rules and select your custom rule that you want {ProductShortName} to use for an analysis. You can use the custom rules in addition to the default rules.
|Configure generative AI|This option opens the `provider-settings.yaml` file that has API keys and other parameters for all supported LLMs. By default, {mta-dl-plugin} uses OpenAI LLM. To change the model, update the anchor `&active` to the desired block. Modify this file with the required arguments, such as the model and API key, to complete the setup.
|====

.Verification

> . Verify that your configuration works by running an analysis:

Reviewer 2: It's a single step with sub-steps. No other main steps follows. Thus, you should use a single bullet:
```suggestion
* Verify that your configuration works by running an analysis:
```

. Open *{ProductShortName} Analysis View*.

> . Click *Run Analysis* to verify that your configuration works by running an analysis.

Reviewer 2: This section doesn't allow standalone sentences, only steps, similarly to the Procedure section:
```suggestion
* Verify that your configuration works by running an analysis:

.. Open *{ProductShortName} Analysis View*.
.. Click *Run Analysis*.
```

|Select profile|Create a profile that you can reuse for multiple analyses. The profile name is part of the context provided to the LLM for analysis.

> |Configure label selector|A label selector filters rules for analysis based on the source or target technology.

Reviewer: Would this be basic configs from the `Get Ready to Analyze` pane? If yes, this should be attached to line 26 with a plus sign.

Author: Yes, I have attached this to line 26 :+1:

> 

Reviewer: Everything after this line is currently floating. They sound like actions so should they continue the procedure list? If so, they must be moved before the verification step. Please verify as DITA doesn't support floating items.
